Add new fields to StorageCluster spec doc (#791)
portworx/pxdocs: content/reference/CRD/storage-cluster.md

| spec.<br>storage.<br>devices[] | Specifies the list of devices Portworx should use. | `[]string` | None |
| spec.<br>storage.<br>journalDevice | Specifies the device Portworx uses for journaling. | `string` | None |
| spec.<br>storage.<br>systemMetadataDevice | Indicates the device Portworx uses to store metadata. For better performance, specify a system metadata device when using Powrtworx with the internal KVdb. | `string` | None |

> | spec.<br>storage.<br>kvdbDevice | Indicates the device Portworx uses to store internal KVdb data. | `string` | None |

Reviewer: ```suggestion
| spec.<br>storage.<br>kvdbDevice | Specifies the device Portworx uses to store internal KVDB data. | `string` | None |
```

| spec.<br>cloudStorage.<br>deviceSpecs[] | A list of the specs for your cloud storage devices. Portworx creates a cloud disk for every device. | `[]string` | None |
| spec.<br>cloudStorage.<br>journalDeviceSpec | Specifies the cloud device Portworx uses for journaling. | `string` | None |
| spec.<br>cloudStorage.<br>systemMetadataDeviceSpec | Indicates the cloud device Portworx uses for metadata. For performance, specify a system metadata device when using Portworx with the internal KVdb. | `string` | None |

> | spec.<br>cloudStorage.<br>kvdbDeviceSpec | Indicates the cloud device Portworx uses for internal KVdb. | `string` | None |

Reviewer: Same question as above

```suggestion
| spec.<br>cloudStorage.<br>kvdbDeviceSpec | Specifies the cloud device Portworx uses for an internal KVDB. | `string` | None |
```

| Field | Description | Type | Default |
| --- | --- | --- | --- |
| spec.<br>placement.<br>nodeAffinity | Use this field to restrict Portwox on certain nodes. It works similarly to the [Kubernetes node affinity](https://github.com/kubernetes/api/blob/master/core/v1/types.go#L2692) feature. | `object` | None |

> | spec.<br>placement.<br>tolerations[] | List of tolerations that will be applied to Portworx pods so that they can run on nodes with matching taints.| `[]object` | None |

Reviewer: Same question about ambiguity. 

```suggestion
| spec.<br>placement.<br>tolerations[] | This field displays a list of tolerations that will be applied to Portworx pods so that they can run on nodes with matching taints.| `[]object` | None |
```

```suggestion
| spec.<br>placement.<br>tolerations[] | This field specifies a list of tolerations that will be applied to Portworx pods so that they can run on nodes with matching taints.| `[]object` | None |
```
